qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -864,6 +864,10 @@ def main():
         # about. Prints to stderr as we can't really count on anything
         # else
         traceback.print_exc(file=ORIGINAL_STDERR)
+        print_file_path = os.path.join(ConfigService.getAppDataDirectory(), 'workbench_stacktrace.txt')
```

Comment:
Couple of things:
* can we define the string `'workbench_stacktrace.txt'` using a named constant put at the top of the module with the others?
* we need to be careful with file handling as it's entirely possible to get an unexpected `OSError`. Can we wrap this in a `try...except` and also use `with open ...` so we don't need to worry about `.close()`